pkg/module/git/source.go

```diff
@@ -1,113 +0,0 @@
-package git
```

Comment:
Any reason why the package needs to be changed? This code is ok and only needs to be either refactored a bit internally and only be called if module dir contains `.git`